src/ompl/tools/thunder/Thunder.h

```diff
@@ -88,13 +88,13 @@ namespace ompl
         public:
             /** \brief Constructor needs the state space used for planning. */
             explicit Thunder(const base::SpaceInformationPtr &si, double stretch_factor = 1.2, double DenseD = 0.001,
-                             double SparseD = 0.1);
+                             double SparseD = 0.04);
```

Comment:
i think we should change this in our call, not in the default here

Reply:
I can take care of that...